student/src/test/java/tds/student/services/ResponseServiceTest.java

```diff
@@ -204,44 +204,6 @@ public void testGetItemGroupFailure () throws SQLException, ReturnStatusExceptio
   // Sucess Test Case
   @Test
   @Ignore("Never actually implemented")
-  public void testisTestComplete () throws SQLException, ReturnStatusException {
```

Comment:
Chose to just delete these tests rather than make them work since they were already ignored and code that we will hopefully remove in the 2018 release.